app/upgrades/v16/upgrades.go

```diff
@@ -26,6 +27,11 @@ func CreateUpgradeHandler(
 		// Enable ICA controller
 		keepers.ICAControllerKeeper.SetParams(ctx, icacontrollertypes.DefaultParams())
 
+		// Set default blocks per epoch
+		providerParams := keepers.ProviderKeeper.GetParams(ctx)
```

Comment:
This will panic since `BlocksPerEpoch` key doesn't exist in the paramstore at this point.

Reply:
The upgrade code is in https://github.com/cosmos/interchain-security/pull/1757

It will should be available in the next release candidate of ICS@v4.1.x